starknet/src/errors.ts

```diff
@@ -0,0 +1,6 @@
+export class ContractError extends Error {
```

Comment:
nit: It would be easier to read if the known error codes were declared in the same file. Also, we can give the object a shorter name, like ERR_CODES, and that'll make the `throw new ContractError(code...` lines format better:

```typescript
throw new ContractError(ERR_CODES.INVALID_CONTRACT_GROUP,
```